### PR TITLE
Add benchmark for TF

### PR DESCRIPTION
**Pull request by raikonenfnu**
-Refactor SharkBenchmarker to run TF
-Add example and test to benchmark TF

### CODE REVIEW

File: setup_venv.sh [hunk 1]
```diff
@@ -64,6 +65,7 @@ elif [[ $(uname -s) = 'Linux' ]]; then
   echo "${Yellow}Linux detected"
   if [ "$PYTHON_VERSION_X_Y" == "3.7" ] || [ "$PYTHON_VERSION_X_Y" == "3.8" ]  || [ "$PYTHON_VERSION_X_Y" == "3.9" ] || [ "$PYTHON_VERSION_X_Y" == "3.10" ] ; then
     torch_mlir_bin=true
+    install_tensorflow=true
```
Comment (powderluv) [inline setup_venv.sh:68]: I think requirements-importer.txt should already bring in tf no ?

File: setup_venv.sh [hunk 2]
```diff
@@ -72,6 +74,17 @@ fi
 # Upgrade pip and install requirements.
 $PYTHON -m pip install --upgrade pip || die "Could not upgrade pip"
 $PYTHON -m pip install --upgrade -r "$TD/requirements.txt" --extra-index-url https://download.pytorch.org/whl/nightly/cpu -f https://github.com/llvm/torch-mlir/releases
+if [ "$install_tensorflow" = true ]; then
+  $PYTHON -m pip install tensorflow
+  $PYTHON -m pip install --find-links https://github.com/google/iree/releases iree-tools-tf
```
Comment (powderluv) [inline setup_venv.sh:79]: this should already happen on linux

Comment (raikonenfnu) [reply setup_venv.sh:79]: got it!

File: setup_venv.sh (hunk 1 above)
Comment (raikonenfnu) [reply setup_venv.sh:68]: yeap, you're right

Comment (raikonenfnu) [reply setup_venv.sh]: done!

File: setup_venv.sh (hunk 2 above)
Comment (raikonenfnu) [reply setup_venv.sh]: done!

**Review by powderluv — APPROVED** [review]

**Pull request merged**